src/commands/uplink/LastEvent.ts

```diff
@@ -0,0 +1,270 @@
+import Command from '../../Command.js';
+import * as hardwareTypes from '../../constants/hardwareTypes.js';
+import CommandBinaryBuffer from '../../CommandBinaryBuffer.js';
+import * as bitSet from '../../utils/bitSet.js';
+import {UPLINK} from '../../constants/directionTypes.js';
+
+
+interface ILastEventGasStatus {
+    /** the battery voltage has dropped below the set threshold */
+    isBatteryLow?: boolean,
+    /** there is a magnetic field influence */
+    isMagneticInfluence?: boolean,
+    /** button is release (device is unmounted) */
+    isButtonReleased?: boolean,
+    /** the device has detected a loss of connection to the server */
+    isConnectionLost?: boolean
+}
+
+interface ILastEvent2channelStatus {
+    /** the battery voltage has dropped below the set threshold */
+    isBatteryLow?: boolean,
+    /** the device has detected a loss of connection to the server */
+    isConnectionLost?: boolean,
+    /** the first channel is not active */
+    isFirstChannelInactive?: boolean,
```

Comment:
```suggestion
    isFirstChannelActive?: boolean,
```
?

Reply:
nope

`true` - the first channel is NOT active